Connectors/Revit/Speckle.Connectors.RevitShared/Bindings/SelectionBinding.cs

```diff
@@ -26,8 +26,10 @@ ITopLevelExceptionHandler topLevelExceptionHandler
     _topLevelExceptionHandler = topLevelExceptionHandler;
     // POC: we can inject the solution here
     // TODO: Need to figure it out equivalent of SelectionChanged for Revit2020
+#if !REVIT2020 && !REVIT2021 && !REVIT2022
```

Comment:
We were talking about being able to say - is this 2022 onwards for instance, to simplify this

Reply:
We do have this in RHINO, following the standard pattern used for c# frameworks: `RHINO7_OR_GREATER`.

I really suggest we do this bc it allows for setting lower/upper bounds without having to explicitly state all versions.